CocktailsPage.java

```diff
@@ -0,0 +1,170 @@
+    private WebElement veryBerrySlushie;
+
+    @FindBy(xpath = "//h2[contains(text(),'Cabernet Berry-Licious')]")
+    private WebElement cabernetBerryLicious;
+
+    @FindBy(xpath = "//h2[contains(text(),'Blooming Shiraz')]")
+    private WebElement bloomingShiraz;
+
+
+
+    //sparkling wine list
+
+    @FindBy(xpath = "//h2[contains(text(),'Prosecco Pear Punch')]")
+    private WebElement proseccoPearPunch;
+
+    @FindBy(xpath = "//h2[contains(text(),'Pink Lemonade')]")
+    private WebElement pinkLemonade;
+
+    @FindBy(xpath = "//h2[contains(text(),'Sorbet Prosecco')]")
+    private WebElement sorbetProsecco;
+
+    @FindBy(xpath = "//h2[contains(text(),'Cranberry Christmas Crush')]")
+    private WebElement cranberryChristmasCrush;
+
+    @FindBy(xpath = "//h2[contains(text(),'Bumble Bee Bevvy')]")
+    private WebElement bumbleBeeBevvy;
+
+    @FindBy(xpath = "//h2[contains(text(),'Blackberry Lavendar Bubble')]")
+    private WebElement blackberryLavendarBubbles;
+
+    @FindBy(xpath = "//h2[contains(text(),'Green Thumb')]")
+    private WebElement greenThumb;
+
+    @FindBy(xpath = "//h2[contains(text(),'Prosecco Plum Cocktail')]")
+    private WebElement proseccoPlumCocktail;
+
+    @FindBy(xpath = "//h2[contains(text(),'Tea Time Spritzer')]")
+    private WebElement teaTimeSpritzer;
+
+    @FindBy(xpath = "//h2[contains(text(),'Fig & Sprig Spritz')]")
+    private WebElement figSprigSpritz;
+
+    @FindBy(xpath = "//h2[contains(text(),'Pink Paradise')]")
+    private WebElement pinkParadise;
+
+
+
+    public CocktailsPage (WebDriver webDriver) { this.webDriver = webDriver;
+        PageFactory.initElements(webDriver,this);}
+
+    public WebElement WineSelect(){
+        return wineSelect;
+    }
+
+    public void clickToSelectWine(){
+        wineSelect.click();
+    }
+
+    public void selectRedWine(){
+        WebDriverWait wait = new WebDriverWait(webDriver,10);
+        wait.until(ExpectedConditions.visibilityOf(redWineSelect));
+        redWineSelect.click(); }
+
+    public void selectSparklingWine(){
+        WebDriverWait wait = new WebDriverWait(webDriver,10);
+        wait.until(ExpectedConditions.visibilityOf(sparklingWineSelect));
+        sparklingWineSelect.click();
+    }
+
+    public void assertAllRedCocktailsAreDisplayed() {
+
+        List<WebElement> listOfAllCocktails = new ArrayList<WebElement>();
+
+        listOfAllCocktails.add(mulledWine);
+        listOfAllCocktails.add(redWineSpritzer);
+        listOfAllCocktails.add(redHotChocolate);
+        listOfAllCocktails.add(yellowTailshake);
+        listOfAllCocktails.add(bloomingShiraz);
+        listOfAllCocktails.add(cabernetBerryLicious);
+        listOfAllCocktails.add(veryBerrySlushie);
+        listOfAllCocktails.add(winterRedSpritz);
+        listOfAllCocktails.add(chocolateFondue);
+
+        for (WebElement element : listOfAllCocktails) {
+            Assert.assertTrue(element.isDisplayed());

```

Comment:
Move assert to Test.
Here verification can be simplified to verify quantity of coctails.
So you don't need to write locators for each coctail.
It will be enough to write just one locator for all of them(List<WebElement> as type of variable)